packages/pytest-simcore/src/pytest_simcore/helpers/utils_postgres.py

```diff
@@ -67,3 +65,40 @@ def is_postgres_responsive(url) -> bool:
     except OperationalError:
         return False
     return True
+
+
+async def _insert_and_get_row(
+    conn, table: sa.Table, values: dict[str, Any], pk_col: sa.Column, pk_value: Any
+):
+    result = await conn.execute(table.insert().values(**values).returning(pk_col))
+    row = result.first()
+
+    # NOTE: DO NO USE row[pk_col] since you will get a deprecation error (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
+    assert getattr(row, pk_col.name) == pk_value
```

Comment:
this one is a bit weird

Reply:
@sanderegg `pk_col` is `sa.Column`. In the old version you could do `row[pk_col] ` but this dict-like interface has been deprecated in the new sqlaclhemy.